src/transformers/training_args.py

```diff
@@ -548,6 +552,12 @@ class TrainingArguments:
         default=False,
         metadata={"help": "Whether to use full float16 evaluation instead of 32-bit"},
     )
+    tf32: bool = field(
+        default=True,
```

Comment:
The default should be whatever PyTorch has by default, so None here and the user can set it to True or False to force/unforce it.

I understand it's True for versions >= 1.7 and < 1.10 but False after?

Reply:
ah, good idea! let the user decide!

Reply:
True for versions >= 1.7 and < 1.11 and probably False after - the nightly is still True as of today.